pkg/storj/redundancy.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+// RedundancyScheme specifies the parameters and the algorithm for redundancy
+type RedundancyScheme struct {
+	Algorithm RedundancyAlgorithm
+
+	ShareSize int64
+
+	RequiredShares int16
+	RepairShares   int16
+	OptimalShares  int16
+	TotalShares    int16
+}
+
+// RedundancyAlgorithm is the algorithm used for redundancy
+type RedundancyAlgorithm byte
+
+// List of supported redundancy algorithms
+const (
+	Invalid = RedundancyAlgorithm(iota)
+	Duplication
```

Comment:
Is this the same as "replication", "mirroring"? Shall we implement it at all?
Note that ReedSolomon with `RequiredShares = 1` should be the same.

Reply:
Yes. Wasn't sure what to call it. Hmm, if ReedSolomon with `RequiredShares = 1`, behaves the same, then it probably makes sense to use it. We can add a func `func Replication(...) RedundancyScheme`, to make it easier to construct.